setup.py

```diff
@@ -16,12 +16,13 @@
 extras = {
     "hdf5": "h5py",
     "zarr": "zarr",
-    "n5": "pyn5"
+    "n5": "pyn5",
+    "cloud": "intern"
```

Comment:
I took the opportunity to add this `extra_install` option, but I think it's fine to omit if you'd rather keep this clean :)

Reply:
All good, can be added to extras here.